tests/integration-tests/tests/common/utils.py

```diff
@@ -159,6 +159,19 @@ def generate_random_string():
     return "".join(random.choice(string.ascii_lowercase + string.digits) for _ in range(16))  # nosec
 
 
+def restart_head_node(cluster):
```

Comment:
Minor: I'd keep it in the test_ephemeral for now.

Reply:
I would like to keep here, next to the `reboot_head_node` function